src/react-components/ory/helpers/node.spec.tsx

```diff
@@ -160,3 +160,75 @@ test("uiTextToFormattedMessage on a list", async ({ mount }) => {
   await expect(component).toContainText("te45pbc0")
   await expect(component).toContainText("q3vvtd4i")
 })
+
+test("button with id 1070007 or 1070008 should have formNoValidate", async ({
+  mount,
+}) => {
+  await test.step("button with id 1070007", async () => {
```

Comment:
Could just be their own tests each

Reply:
i guess, but these button labels are the only ones with the formnovalidate attribute. so i grouped them. Is it just stylistically better to separate them?

Reply:
that's a good point! i'll separate them